### PR TITLE
Backend file structure refactoring

### PR DESCRIPTION
**Pull request by no-comma**
Documentation of code style and structure will be written separately to the wiki.

### CODE REVIEW

File: server/src/app.ts [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
+import cors from 'cors';
 import express, { Application } from 'express';
+
 import { router } from './routes/index';
-import cors from 'cors';
 
 export const app: Application = express();
 
 app.use(cors({
   origin: 'http://localhost:3005',
```
Comment (jj-stigell) [inline server/src/app.ts:13]: instead of hardcoding, should we use the `FRONTEND_ORIGIN` env here?

**Review by jj-stigell — CHANGES_REQUESTED** [review]

File: server/src/app.ts (hunk 1 above)
Comment (no-comma) [reply server/src/app.ts]: Yes, good point.

**Review by jj-stigell — APPROVED** [review]
👍

**Comment by no-comma** [comment]
This could also split `src/routes/index.ts` into multiple files now that the Swagger PR got merged, but that may be better left to API refactoring PRs.

**Review by Tonits — APPROVED** [review]
Looks great!

**Pull request merged**